test/integration_test/operator-test-pod.yaml

```diff
@@ -51,6 +51,7 @@ spec:
     - -t
     - /operator.test
     - -test.v
+    - -test.failfast
```

Comment:
do we really need this?

Reply:
It helps so that if a test fails, we don't keep making more StorageCluster objects (either in case the first one does eventually finish cleaning up, or just to avoid spam). It also helps the tests finish faster if there's some simple setup issue or an early one fails.

Also I _did_ confirm this works unlike that time we added it in PSO tests and it wasn't actually doing anything :joy: